cmd/deploy-with-helm/main.go

```diff
@@ -278,8 +280,9 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
-		gitCommitSHAs[hc.Name] = map[string]string{
-			"gitCommitSha": gitCommitSHA,
+		cliValues = append(cliValues, fmt.Sprintf("--set=%s.image.tag=%s", hc.Name, gitCommitSHA))
+		if releaseName == ctxt.Component {
+			cliValues = append(cliValues, fmt.Sprintf("--set=%s.fullnameOverride=%s", hc.Name, hc.Name))
```

Comment:
For transition from ODS Jenkins I am settting `fullnameOverride` in component `admin` to `admin-helm` so that helm can crate the components side by side with Jenkins. Would it be feasible to somehow enable influencing the parameter set here to honor such an override?

Reply:
> Would it be feasible to somehow enable influencing the parameter set here to honor such an override?

I think it would complicate this task too much.

What you can always do is change the logic of the `chart.fullname` helper. Simply introduce a new value, such as `tmpOverride`, that you set to something you want. Then the logic can favour `tmpOverride` over `fullnameOverride` and you are good to go.